Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -950,433 +906,440 @@
+    <RegEx find="([rR])ešenj(e|em|u|a|ima)?" replaceWith="$1ješenj$2" />
+    <RegEx find="\b([rR])j?eč(i|ima)" replaceWith="$1iječ$2" />
+    <RegEx find="\b([rR])j?eš(io|ila|ili|ile|iti|im|iš|i|imo|ite|e|en|eno|ena|ene|eni)" replaceWith="$1iješ$2" />
+    <RegEx find="sačeka(j|te|š|mo|te|ju|li|še|te|jte)?" replaceWith="pričeka$1" />
+    <RegEx find="Sačeka(j|te|š|mo|te|ju|li|še|te|jte)?" replaceWith="Pričeka$1" />
+    <RegEx find="([sS])aglasno(st|sti|šću|stima)?" replaceWith="$1uglasno$2" />
+    <RegEx find="([sS])amoubistv(a|o|u|om|ima)" replaceWith="$1amoubojstv$2" />
+    <RegEx find="saobraćaj(na|nu|ni|nom|nima)?" replaceWith="promet$1" />
+    <RegEx find="Saobraćaj(na|nu|ni|nom|nima)?" replaceWith="Promet$1" />
+    <RegEx find="([sS])aradni(ka|ku|kom|ca|ce|ci|cu|co|cima|cama)" replaceWith="$1uradni$2" />
+    <RegEx find="([sS])arađ(ivam|ivaš|iva|ivamo|ivate|ivaju|ivati|ujem|uješ|uje|ujemo|ujete|uju|ujući|ujuća|ivanje|ivanjem)?" replaceWith="$1urađ$2" />
+    <RegEx find="([sS])atan(a|e|i|istički)?" replaceWith="$1oton$2" />
+    <RegEx find="\b([sS])avet(a|u|om|ima|nik|nica|nica|nicu|nici|nicima|nicama|nice|ovanje|ovanju)?" replaceWith="$1avjet$2" />
+    <RegEx find="([sS])ažalj?eva(m|š|mo|te|ju|ti|li|le|la|lo|jući|juća|juće|nje)" replaceWith="$1ažalijeva$2" />
+    <RegEx find="\b([sS])edi(m|š|i|imo|ite|e|eći|ili|ile|ilo|iše)\b" replaceWith="$1jedi$2" />
+    <RegEx find="\b([sS])istem(a|e|i|u|om|ima)?\b" replaceWith="$1ustav$2" />
+    <RegEx find="([sS])mećar(a|e|i|ski|ska|sko|skom|skoj|skih)?" replaceWith="$1metlar$2" />
+    <RegEx find="\b([sS])mj?e(m|š|mo|te|šna|šne|šni|šno|šnom|šnoj|šnu|im|ima)\b" replaceWith="$1mije$2" />
+    <RegEx find="\b([sS])mej(u|e|em|eš|emo|ete|anje|anju|ati|ali|ale|ao|ala|alo)" replaceWith="$1mij$2" />
+    <RegEx find="([sS])mest(i|ili|ile|io|ila|ilo|iti|ivši)" replaceWith="$1mjest$2" />
+    <RegEx find="([sS])pasava(m|š|mo|te|ju|li|nje|nju|nja)?" replaceWith="$1pašava$2" />
+    <RegEx find="([sS])pri?j?ečava(m|š|mo|te|ju|li|še|nje)?" replaceWith="$1prječava$2" />
+    <RegEx find="([sS])prj?eči(m|š|mo|te|li|še|o|la|li|le|lo|ti)?" replaceWith="$1priječi$2" />
+    <RegEx find="([sS])prj?eče" replaceWith="$1priječe" />
+    <RegEx find="([sS])umlja(m|š|še|mo|te|ti|ju|jući|li|le|lo)?" replaceWith="$1umnja$2" />
+    <RegEx find="\b([sS])eti(t|o|m|š|la|li|le|ti)?\b" replaceWith="$1jeti$2" />
+    <RegEx find="([sS])ever(a|u|om|ni|nom|ac)?" replaceWith="$1jever$2" />
+    <RegEx find="([sS])eća(m|š|mo|te|ju|li|nje)?" replaceWith="$1jeća$2" />
+    <RegEx find="([sS])hvata(m|š|mo|te|ju|jući|la|li|le|lo|ti|nje)?" replaceWith="$1hvaća$2" />
+    <RegEx find="([sS])led(eće|ećeg|eća|eću|ećom|ećoj|ećem|eći|ećom|ećima|ećim|ećih)?" replaceWith="$1ljed$2" />
+    <RegEx find="\b([sS])ljed(a|e|i|u|om|im|iš|imo|ite|e|ili)?\b" replaceWith="$1lijed$2" />
+    <RegEx find="([rR])edosljed(a|e|u|om)?" replaceWith="$1edoslijed$2" />
+    <RegEx find="([sS])meh(a|u|om)?" replaceWith="$1mijeh$2" />
```

Comment:
Logically, this makes no sense without a `\b` after `)?`, or, alternatively, removing the `?` at the end.

I'm guessing, the intention is
`<RegEx find="([sS])mj?eh(a|u|om)?\b" replaceWith="$1mijeh$2" />`
meaning `eh`, `eha`, `ehu`, or `ehom` will always be at the end of a word.